public/use-cases/ca/political-parties.md

```diff
@@ -69,7 +69,7 @@ A mesura que les organitzacions polítiques s'enfronten a un augment del control
 - Reduir els costos operatius i la càrrega administrativa.  
   En adoptar la votació digital basada en blockchain, els partits polítics poden construir un futur més transparent, segur i democràtic, assegurant que la veu de cada membre sigui escoltada sense compromís.
 
-### Referències:
+### **Referències**
```

Comment:
If you have to do this, it means the styles for headings are not properly set. Headings are (and usually should be) always bold.